tiny11builder.bat

```diff
@@ -0,0 +1,203 @@
+		reg add "HKLM\zDEFAULT\Control Panel\UnsupportedHardwareNotificationCache" /v "SV2" /t REG_DWORD /d "0" /f >nul 2>&1
+		reg add "HKLM\zNTUSER\Control Panel\UnsupportedHardwareNotificationCache" /v "SV1" /t REG_DWORD /d "0" /f >nul 2>&1
+		reg add "HKLM\zNTUSER\Control Panel\UnsupportedHardwareNotificationCache" /v "SV2" /t REG_DWORD /d "0" /f >nul 2>&1
+		reg add "HKLM\zSYSTEM\Setup\LabConfig" /v "BypassCPUCheck" /t REG_DWORD /d "1" /f >nul 2>&1
+		reg add "HKLM\zSYSTEM\Setup\LabConfig" /v "BypassRAMCheck" /t REG_DWORD /d "1" /f >nul 2>&1
+		reg add "HKLM\zSYSTEM\Setup\LabConfig" /v "BypassSecureBootCheck" /t REG_DWORD /d "1" /f >nul 2>&1
+		reg add "HKLM\zSYSTEM\Setup\LabConfig" /v "BypassStorageCheck" /t REG_DWORD /d "1" /f >nul 2>&1
+		reg add "HKLM\zSYSTEM\Setup\LabConfig" /v "BypassTPMCheck" /t REG_DWORD /d "1" /f >nul 2>&1
+		reg add "HKLM\zSYSTEM\Setup\MoSetup" /v "AllowUpgradesWithUnsupportedTPMOrCPU" /t REG_DWORD /d "1" /f >nul 2>&1
+echo Removing Edge and WebView2
+		rd /s /q "%HOMEDRIVE%\scratchdir\Program Files (x86)\Microsoft\Edge" >nul
+		rd /s /q "%HOMEDRIVE%\scratchdir\Program Files (x86)\Microsoft\EdgeUpdate" >nul
+		rd /s /q "%HOMEDRIVE%\scratchdir\Program Files (x86)\Microsoft\EdgeWebView" >nul
+		rd /s /q "%HOMEDRIVE%\scratchdir\Program Files (x86)\Microsoft\EdgeCore" >nul
+		reg delete "HKLM\zSOFTWARE\WOW6432Node\Microsoft\EdgeUpdate" /f >nul 2>&1
+		reg delete "HKLM\zSOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft Edge" /f /v "NoRemove" >nul 2>&1
+		reg delete "HKLM\zSOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft Edge Update" /f /v "NoRemove" >nul 2>&1
+		reg delete "HKLM\zSOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft EdgeWebView" /f /v "NoRemove" >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Microsoft\EdgeUpdate" /f /v InstallDefault /d 0 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Microsoft\EdgeUpdate" /f /v Install{56EB18F8-B008-4CBD-B6D2-8C97FE7E9062} /d 0 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Microsoft\EdgeUpdate" /f /v Install{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5} /d 0 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Microsoft\EdgeUpdate" /f /v DoNotUpdateToEdgeWithChromium /d 1 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Policies\Microsoft\EdgeUpdate" /f /v InstallDefault /d 0 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Policies\Microsoft\EdgeUpdate" /f /v Install{56EB18F8-B008-4CBD-B6D2-8C97FE7E9062} /d 0 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Policies\Microsoft\EdgeUpdate" /f /v Install{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5} /d 0 /t reg_dword >nul 2>&1
+		reg add "HKLM\zSOFTWARE\Policies\Microsoft\EdgeUpdate" /f /v DoNotUpdateToEdgeWithChromium /d 1 /t reg_dword >nul 2>&1
+echo Removing OneDrive
+		takeown /f %HOMEDRIVE%\scratchdir\Windows\System32\OneDriveSetup.exe >nul
+		icacls %HOMEDRIVE%\scratchdir\Windows\System32\OneDriveSetup.exe /grant *S-1-5-32-544:F /T /C >nul

```

Comment:
Remove the `/T` here. Without it it causes icacls to recurse through the entire System32 folder. Also it causes two error messages:
```
C:\scratchdir\Windows\System32\config\systemprofile\AppData\Local\Microsoft\Windows\INetCache\Content.IE5\*: Access is denied.
C:\scratchdir\Windows\System32\LogFiles\WMI\RtBackup\*: Access is denied.
C:\scratchdir\Windows\System32\WebThreatDefSvc\*: Access is denied.
```